javascript/javascript3/week2/js3Week3Ex.js

```diff
@@ -0,0 +1,151 @@
+class user{
```

Comment:
I think this exercise doesn't bellong to the week2 pull request. You can remove the fille and push another commit for the week2